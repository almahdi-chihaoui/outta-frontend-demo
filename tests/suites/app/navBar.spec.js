import React from "react"; 
import renderer from "react-test-renderer";
import NavBar from "../../../src/containers/app/navBar";

test('NavBar component renders with h1 text', () => {
    const component = renderer.create(<NavBar></NavBar>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});