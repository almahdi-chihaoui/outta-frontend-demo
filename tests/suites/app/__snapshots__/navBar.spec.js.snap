// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component renders with h1 text 1`] = `
<div>
  <header
    className="MuiPaper-root-10 MuiPaper-elevation4-16 MuiAppBar-root-1 MuiAppBar-positionStatic-5 MuiAppBar-colorPrimary-8"
  >
    <div
      className="MuiToolbar-root-37 MuiToolbar-regular-39 MuiToolbar-gutters-38"
    >
      <h6
        className="MuiTypography-root-41 MuiTypography-h6-58 MuiTypography-colorInherit-70"
      >
        Outta
      </h6>
    </div>
  </header>
</div>
`;
